__mocks__/react-i18next.ts

```diff
@@ -1,4 +1,7 @@
 module.exports = {
   useTranslation: () => ({ t: (key) => key }),
   Trans: ({ children }) => children,
+  getI18n: () => ({
```

Comment:
why we need it?

Reply:
For tests we mock react-i18next and they fails because the script does not find that function.
Tests started failing when i imported the `t` from the utils in the constants file